src/components/calculator.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] Kindly avoid disabling linters, instead I recommend you learn how to work around them since it's a project requirement in every Microverse project and also they force us to write code that follows best practices 👍